compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/InteractionUIView.uikit.kt

```diff
@@ -201,4 +507,15 @@ internal class InteractionUIView(
         inInteractionBounds = { false }
         onKeyboardPresses = {}
     }
+
+    /**
+     * Execute the given [hitTestBlock] and save the result to the gesture recognizer handler, so
+     * that it can be used later to determine if the gesture recognizer should be recognized
+     * or failed.
+     */
+    private fun rememberHitTestResult(hitTestBlock: () -> UIView?): UIView? {
```

Comment:
`rememberX` has very specific semantic in Compose, please rename to avoid confusions

Reply:
Done